### PR TITLE
Release 9.6.0

### PR DESCRIPTION
**Pull request by clairesunstudio**
### Added
- (Patternlab) [SectionLinks] DP-13100: Changed topic card heading styles, and footer CTA styles #542
- (React) [InputTextFuzzy] DP-13688: Added option to render all options as suggestions by default, via prop `renderDefaultSuggestion` #544
- (React) [Storybook Config] DP-13688: Added storybook console addon #544

### Removed
- (Patternlab) [assets] DP-12475: Remove richText.js. #505
- (Patternlab) [RichText] Removed js flag classes for richText.js. #505

### Fixed
- (React) [FilterBox] Fixed warning caused by using `start` instead of `flex-start` within stying. #543

### CODE REVIEW

**Review by mrossi113 — APPROVED** [review]
👍 LGTM!

File: assets/scss/00-base/_colors.scss [hunk 1]
```diff
@@ -25,6 +25,7 @@ $c-granite-gray: #535353;
 $c-overcast-gray: #DCDCDC;
 $c-platinum-gray: #F2F2F2;
 $c-atlantic-gray: #707070;
+$c-very-light-gray: #CCCCCC;
```
Comment (avrilpearl) [inline assets/scss/00-base/_colors.scss:28]: we should look at this - it should be added to the gray variable color scale in this file as well.

**Review by avrilpearl — APPROVED** [review]
approved - just want to follow up on that color variable addition/change

File: assets/scss/00-base/_colors.scss (hunk 1 above)
Comment (clairesunstudio) [reply assets/scss/00-base/_colors.scss:28]: good catch, so the structure of base color variables are not meant to be used direction, we want to pass these to a greyscale variable. Let's do follow up later

**Pull request merged**